includes/Modules/Analytics/Advanced_Tracking/Event_List_Registry.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Event_List_Registry
+ *
+ * @package   Google\Site_Kit\Modules\Analytics
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Analytics\Advanced_Tracking;
+
+use Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Measurement_Events\Measurement_Event_List;
+
+/**
+ * Class for registering third party event lists.
+ *
+ * @since n.e.x.t.
+ * @access private
+ * @ignore
+ */
+class Event_List_Registry {
+
+	/**
+	 * The list of active event lists.
+	 *
+	 * @since n.e.x.t.
+	 * @var Measurement_Event_List[]
+	 */
+	private $active_event_lists;
+
+	/**
+	 * Event_List_Registry constructor.
+	 *
+	 * @since n.e.x.t.
+	 */
+	public function __construct() {
+		$this->active_event_lists = array();
+	}
+
+	/**
+	 * Registers a third party event lists.
+	 *
+	 * @since n.e.x.t.
+	 *
+	 * @param Measurement_Event_List $event_list The third party event list to be registered.
```

Comment:
No need to mention "third party" here, it can be any event list - Site Kit could register one too (which is technically what's happening now).

Reply:
Resolved.